recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
From https://conda.io/docs/building/recipe.html#conda-recipe-files-overview, the building takes place in the source directory. `$PREFIX` is the location defined by conda-build to which the build script should install executables (https://conda.io/docs/building/environment-vars.html#environment-variables-set-during-the-build-process). In simple cases, this is:

```bash
cp executable $PREFIX/bin
```